tests/sb-util.test.ts

```diff
@@ -134,5 +141,44 @@ describe('Sprite class -------------------------', () => {
 	test('can get lists', () => {
 		const lists = sprite.lists();
 		expect(lists).toBeInstanceOf(Object);
-	})
+	});
+
+	test('can get all blocks', () => {
+		const blocks = sprite.blocks();
+		expect(blocks).toBeInstanceOf(BlockCollection);
+	});
+});
+
+describe('BlockCollection class', () => {
+	let sp, blocks;
+
+	beforeAll(async () => {
+		// Choosing this file because it has a lot of blocks
+		sp = await loadProjectJson(`${process.cwd()}/tests/data/accelerator.json`);
+		blocks = sp.blocks();
+	});
+
+	test('query for opcode', () => {
```

Comment:
Do we need to test that the collection iterators Block instances?

Reply:
That would be a good test! I will add it